tutorials/colab-env-setup.ipynb

```diff
@@ -74,15 +74,6 @@
     "Pick one of the following datasets to download for the appropriate tutorial you will work on"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "!mkdir /root/brainiak_datasets"
```

Comment:
So my reasoning for having this stand-alone mkdir was that someone might try to download the data before the installation. It happened to me once.
We could change it to check if the directory already exists, and only run mkdir if it doesn't.

Reply:
Updated mkdir with -p flag.